presto-main/src/main/java/com/facebook/presto/operator/OperatorStats.java

```diff
@@ -46,6 +46,8 @@
     private final long addInputCalls;
     private final Duration addInputWall;
     private final Duration addInputCpu;
+    private final DataSize rawInputDataSize;
+    private final long rawInputPositions;
```

Comment:
Do we need both, `rawInputPositions` and `inputPositions`?

Reply:
@sopel39  are these always the same value?